packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -28,6 +28,14 @@ const Color mainCpuColor = Color(0xFF64B5F6);
 // https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
 const Color mainGpuColor = Color(0xFF4DB6AC);
```

Comment:
Add a TODO to switch these from Color to ThemedColor so these will work with the Dark theme as well.

Reply:
thx. done.